chromium_src/chrome/browser/flags/android/chrome_feature_list.cc

```diff
@@ -62,6 +62,7 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kAndroidHub, base::FEATURE_DISABLED_BY_DEFAULT},
     {tab_groups::kAndroidTabGroupStableIds, base::FEATURE_DISABLED_BY_DEFAULT},
     {kIncognitoReauthenticationForAndroid, base::FEATURE_ENABLED_BY_DEFAULT},
+    {kMagicStackAndroid, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
if we have our own NTP then does this flag actually do anything?

Reply:
cc: @samartnik

Reply:
It adds setting to the Settings menu that we don't need